src/components/__tests__/__snapshots__/autocomplete.spec.tsx.snap

```diff
@@ -30,7 +30,9 @@ exports[`Autocomplete component should render 1`] = `
     >
       <div
         class="dropdown-menu__panel"
-      />
+      >
+        <ul />
+      </div>
```

Comment:
Probably shouldn't render an empty list. Not sure if it's just because the data used for this test is not realistic and this won't happen in the real world

Reply:
changed